UIKitTemplate/UIKitTemplate/Controllers/TabBarViewController.swift

```diff
@@ -0,0 +1,73 @@
+// TabBarViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+// Tab Bar Controller
```

Comment:
```suggestion
// Tab Bar Controller
```
Коменты только на русском пиши везде а то тут так там так